### ISSUE TITLE
32-bit builds of 0.9.5.1 fail C function test

### ISSUE THREAD

**Issue by clinty** [hslua/hslua#63]
32-bit builds of 0.9.5.1 fail C function test:

```
    C functions
      Registering a C function and calling it from Lua:           FAIL
        test/Foreign/LuaTest.hs:162:
        greeting function failed
        expected: Right ["Caffeine","induced","nonsense"]
         but got: Right []
```

https://buildd.debian.org/status/fetch.php?pkg=haskell-hslua&arch=i386&ver=0.9.5.1-3&stamp=1524238981&raw=0

**Comment by tarleb** [comment]
Thanks for the report and apologies for the late reply. I'm a bit puzzled by this error. I noted that the build is currently not using the *lua_32bits* cabal flag. Could you try building with that flag?

**Comment by tarleb** [comment]
Tests succeed now on 32 Windows (AppVeyor), so it seems save to close this. Fix will be released with hslua-1.0.0.
